x-pack/test/functional/page_objects/status_page.ts

```diff
@@ -5,36 +5,18 @@
  * 2.0.
  */
 
-import expect from '@kbn/expect';
 import { FtrProviderContext } from '../ftr_provider_context';
 
 export function StatusPagePageProvider({ getService }: FtrProviderContext) {
-  const retry = getService('retry');
   const log = getService('log');
-  const browser = getService('browser');
   const find = getService('find');
-  const deployment = getService('deployment');
```

Comment:
This test won't work on Cloud: user can't navigate to `/status` page after log out. So I think we can safely remove deployment service usage.